packaging/nnstreamer.spec

```diff
@@ -244,6 +263,9 @@ cp -r result %{buildroot}%{_datadir}/nnstreamer/unittest/
 %endif
 
 %post
+pushd %{_prefix}/lib
+cp nnstreamer/filters/nnstreamer_python2.so python2.7/site-packages/
```

Comment:
When the Tizen team will try to change the Python version from 2.7 to 2.x (e.g. 2.8), What will be happened to us? :)
Can we find out better approach? 
* before: python2.7/site-packages/
* after : %{_python_version}/site-packages/

Reply:
Don't duplicate files in embedded systems!

Reply:
Sorry for the late reply.
@leemgs  Updated to %{python_sitelib}. In fact, python 2.7 is equivalent to python 2. On the other hand, python 3 has many different versions (e.g., 3.5, 3.6, ...). 
@myungjoo  Got it.

Reply:
Ok. LGTM (except for re-stating default-true values: -Denable-python=true )